code/modules/tutorials/flock_tutorial.dm

```diff
@@ -0,0 +1,449 @@
+		..()
+		var/turf/T = locate(src.ftutorial.center.x, src.ftutorial.center.y - 6, src.ftutorial.center.z)
+		var/obj/deployable_turret/riot/turret = new(T)
+		turret.set_angle(0)
+		turret.set_projectile()
+		sleep(1 SECOND)
+		muzzle_flash_any(turret, 0, "muzzle_flash")
+		shoot_projectile_ST_pixel_spread(turret, turret.current_projectile, src.ftutorial.center, 0, 0 , turret.spread)
+		SPAWN(10 SECONDS)
+			src.ftutorial.Advance()
+
+/datum/tutorialStep/flock/relay
+	name = "The Relay"
+	New()
+		src.instructions = "This is the Relay, your ultimate goal. Unlocked at [FLOCK_RELAY_COMPUTE_COST] compute, when complete it will allow you to transmit the Signal and cast the Flock out towards the Source."
+		..()
+
+	SetUp()
+		..()
+		var/turf/T = get_turf(src.ftutorial.center)
+		for (var/obj/flock_structure/cage/cage in range(3, T))
+			qdel(cage.occupant)
+			qdel(cage)
+		var/obj/flock_structure/ghost/tealprint = new(T, src.ftutorial.fowner.flock, /obj/flock_structure/relay)
+		tealprint.fake = TRUE
+
+/datum/tutorialStep/flock/showcase
+	name = "Structure showcase"
+	instructions = "Here are all the Flock structures you can create, along with a shadow of the Relay, your ultimate goal. Click the exit tutorial button in the bottom right corner to exit the tutorial."
+	SetUp()
+		..()
+		src.ftutorial.fowner.abilityHolder.addAbility(/datum/targetable/flockmindAbility/tutorial_exit)
+		var/datum/mapPrefab/allocated/prefab = get_singleton(/datum/mapPrefab/allocated/flock_showcase)
+		var/datum/allocated_region/region = prefab.load()
+		for (var/turf/T in REGION_TILES(region))
+			var/obj/spawner/flock_structure/structure_spawner = locate() in T
+			structure_spawner?.spawn_structure(ftutorial.fowner.flock)
+			var/mob/living/carbon/human/bad_immortal/fake_tdummy = locate() in T
+			if (fake_tdummy)
+				src.ftutorial.fowner.flock.updateEnemy(fake_tdummy)
+		var/turf/landmark = null
+		for(var/turf/T as anything in landmarks[LANDMARK_TUTORIAL_START])
+			if(region.turf_in_region(T))
+				landmark = T
+				break
+		if (!landmark)
+			CRASH("No flock showcase landmark found, aaaaa")
+		src.ftutorial.fowner.set_loc(landmark)
+		src.ftutorial.ui_interact(src.ftutorial.fowner)
+		for (var/mob/living/intangible/flock/trace/trace as anything in src.ftutorial.fowner.flock.traces)
+			trace.set_loc(get_turf(src.ftutorial.fowner))
+			src.ftutorial.ui_interact(trace)
+
+/mob/living/intangible/flock/flockmind/verb/help_my_tutorial_is_being_a_massive_shit()
+	set name = "EMERGENCY TUTORIAL STOP"
+	if (!src.tutorial)
+		boutput(src, "<span class='alert'>You're not in a tutorial, doofus. It's real. IT'S ALL REAL.</span>")
+		return
+	src.tutorial.Finish()
+	src.tutorial = null
+
+/mob/living/critter/flock/drone/verb/help_my_tutorial_is_being_a_massive_shit()
+	set name = "EMERGENCY TUTORIAL STOP"
+	if (istype(src.controller, /mob/living/intangible/flock/flockmind))
+		var/mob/living/intangible/flock/flockmind/flockmind = src.controller
+		src.release_control()
```

Comment:
release_control() moves the controller back to the station right now if not on station z level, would happen again with tutorial finish but not sure if noticeable at all

Reply:
Should put them in the same spot so 🤷